curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/write-higher-order-arrow-functions.english.md

```diff
@@ -51,14 +79,20 @@ tests:
 const realNumberArray = [4, 5.6, -9.8, 3.14, 42, 6, 8.34, -2];
 const squareList = (arr) => {
   "use strict";
-  // change code below this line
-  const squaredIntegers = arr;
-  // change code above this line
+  const positiveIntegers = arr.filter((num) => {
+     // add code here
+  });
+  const squaredIntegers = positiveIntegers.map((num) => {
+       // add code here
+});
+
+  // add code here
```

Comment:
I do not think this line should be here, because it implies something else should be added when the only thing left to do is `return squaredIntegers`.
```suggestion

```

Reply:
@RandellDawson I applied those changes using the windows here on Github. They were legit. Thanks for catching those.